saleor/plugins/manager.py

```diff
@@ -2120,7 +2120,9 @@ def _get_channel_map(self):
 
 def get_plugins_manager(
     requestor_getter: Optional[Callable[[], "Requestor"]] = None,
-    allow_replica=True,
+    allow_replica: Optional[bool] = None,
```

Comment:
Why does it default to `None` instead of being required? You should be able to change the signature to:

```python
def get_plugins_manager(
    /,
    allow_replica: bool,
    requestor_getter: Optional[Callable[[], "Requestor"]] = None,
) -> PluginsManager:
    ...
```

or:

```python
def get_plugins_manager(
    requestor_getter: Optional[Callable[[], "Requestor"]] = None,
    /,
    allow_replica: bool,
) -> PluginsManager:
    ...
```

Reply:
fix: https://github.com/saleor/saleor/pull/15003/